src/olympia/files/decorators.py

```diff
@@ -54,7 +54,7 @@ def last_modified(request, obj, key=None, **kw):
 
 
 def etag(request, obj, key=None, **kw):
-    return _get_value(obj, key, 'md5')
+    return _get_value(obj, key, 'sha256')
```

Comment:
Adding @jasonthomas so he is aware. Note that this is limited to file-browsing pages only which is currently accessible to reviewers and admins only, not to the public.